ui/components/component-library/text/text.test.js

```diff
@@ -218,4 +218,9 @@ describe('Text', () => {
       'text--text-transform-capitalize',
     );
   });
+  it('should accept a ref prop that is passed down to the html element', () => {
+    const mockRef = jest.fn();
+    render(<Text ref={mockRef} />);
+    expect(mockRef).toHaveBeenCalledTimes(1);
+  });
```

Comment:
I don't think this is the most robust test but I struggled to find a reliable way to unit test forwarding refs open to suggestions